### DISCUSSION TITLE
Issue report an blocked

### DISCUSSION THREAD

**Discussion by nmm71** [community/community#128867] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

in Nov 23 I sent an issue, see:

https://github.com/MiKTeX/miktex-packaging/issues/413#issue-1994474508

and it was closed from the pepository owner without a usefull answer. 

And now I can not post further issues, because he seems to block me to... I dont understand...

I can not follow the hint "contact the repository owner", because I could not find contact-infos for him ([edocevoli]) in the mixtex-site.

I would like to reopen the issue, becase the problem still exists, also in the newest version 24.1.

What should I do?
Regards
MM

**Comment by ebndev** [comment]
Hi @nmm71,

I see in the issue that the owner said that they are unable to fix the issue.

This is a public forum, and the community is unable to assist with your issue. I'm not sure if there's anything you can do if the owners do not want to work on the feature and decided to block you. Sorry 🫤

As no other Community Discussions team members will be able to provide additional, I am going to close this discussion.
